agent/command/git.go

```diff
@@ -186,17 +192,25 @@ func (opts cloneOpts) getCloneCommand() ([]string, error) {
 	case "", evergreen.CloneMethodLegacySSH:
 		return opts.buildSSHCloneCommand()
 	case evergreen.CloneMethodOAuth:
-		return opts.buildHTTPCloneCommand()
+		return opts.buildHTTPCloneCommand(false)
+	case evergreen.CloneMethodAccessToken:
+		return opts.buildHTTPCloneCommand(true)
 	}
 	return nil, errors.New("unrecognized clone method in options")
 }
 
-func (opts cloneOpts) buildHTTPCloneCommand() ([]string, error) {
+func (opts cloneOpts) buildHTTPCloneCommand(forApp bool) ([]string, error) {
 	urlLocation, err := url.Parse(opts.location)
 	if err != nil {
 		return nil, errors.Wrap(err, "parsing URL from location")
 	}
-	clone := fmt.Sprintf("git clone %s '%s'", thirdparty.FormGitUrl(urlLocation.Host, opts.owner, opts.repo, opts.token), opts.dir)
+	gitUrl := ""
```

Comment:
nit: some people prefer to use `var gitURL string` to indicate that the empty string is not a meaningful value, and will be actually initialized later on (i.e. in the if-else)